src/rules/declaration-block-no-ignored-properties/index.js

```diff
@@ -0,0 +1,104 @@
+import { vendor } from "postcss"
+import {
+  report,
+  ruleMessages,
+  validateOptions,
+  matchesStringOrRegExp,
+} from "../../utils"
+
+export const ruleName = "declaration-block-no-ignored-properties"
+
+export const messages = ruleMessages(ruleName, {
+  rejected: (p, v) => `Unexpected ignored property "${p}" with "${v}"`,
```

Comment:
Can we give these arguments meaningful names, like `ignored` and `cause`?

Also, how about `Unexpected property "${ignored}" that is ignored because of "${cause}"`?